### PR TITLE
log/shell: Add support for logging cbor logs in shell

### PR DESCRIPTION
**Pull request by kolodgrz**
This patch adds possibility to pretty print cbor logs in log shell.

### CODE REVIEW

**Comment by kolodgrz** [comment]
@andrzej-kaczmarek @rymanluk @michal-narajowski

File: sys/log/full/src/log_shell.c [hunk 1]
```diff
@@ -44,6 +47,12 @@ shell_log_dump_entry(struct log *log, struct log_offset *log_offset,
     char data[128];
```
Comment (vrahane) [inline sys/log/full/src/log_shell.c]: This kind of restricts the data length for the cbor entry. You might want to be able to read a longer indefinite length entry.

**Comment by sterlinghughes** [comment]
can we update, so we can merge?

File: sys/log/full/src/log_shell.c (hunk 1 above)
Comment (kolodgrz) [reply sys/log/full/src/log_shell.c]: @vrahane this is not a change in this PR. I can add separate patch to this PR with adoption of this log data size - @vrahane have any idea how big it should be ?

**Comment by andrzej-kaczmarek** [comment]
fixed in https://github.com/apache/mynewt-core/pull/1548

**Pull request closed without merging**